### PR TITLE
Added script docs and simplified script

### PR DESCRIPTION
**Pull request by microman69**

### CODE REVIEW

**Comment by microman69** [comment]
Adding more doc before final PR

File: Source/AzureSqlBulkFailover.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+## Usage
+
+This script is intended to be run as an Azure Automation Runbook or locally.
+
+## Parameters
+
+- `SubscriptionId`: The ID of the subscription to failover. Optional.
+- `ResourceGroupName`: The name of the resource group to failover. Optional.
+- `LogicalServerName`: The name of the logical server to failover. Optional.
```
Comment (bartduncan) [inline Source/AzureSqlBulkFailover.md:13]: I think we should document how RG and LSN combine/interact with each other. 

If one is null and the other is non-null, is only the non-null value used? 

If both are non-null, are they combined with Boolean AND, or OR?

File: Source/AzureSqlBulkFailover.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+## Notes
+
+- This script will failover ALL resources that the caller has access to in all subscriptions in the tenant, filtering by SubscriptionId, ResourceGroupName and LogicalServerName.
```
Comment (bartduncan) [inline Source/AzureSqlBulkFailover.md:17]: "Caller" should be clarified. Maybe "that the service principal that executes the runbook has access to"?

File: Source/AzureSqlBulkFailover.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+- The base URI for ARM API calls is `https://management.azure.com`.
+- The script will wait for 15 seconds on each itteration before checking failover status.
```
Comment (bartduncan) [inline Source/AzureSqlBulkFailover.md:19]: typo (single 't')

File: Source/AzureSqlBulkFailover.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+## Enumerations, globals and helper functions
+
+- `FailoverStatus`: An enumeration containing resource object FailoverStatus values.
+    - `Pending`: The resource is pending to be failedover.
+    - `InProgress`: The failover process is in progress.
+    - `Succeeded`: The failover process succeeded.
+    - `Skipped`: The failover process was skipped (the database did not need to be failed over or was not eligible).
```
Comment (bartduncan) [inline Source/AzureSqlBulkFailover.md:27]: I don't see Skipped used... can it be removed from the script? (probably in a different PR, it's not related to your goals for this one)

File: Source/RunbookEntrypoint.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+GitHub Copilot: Sure, here's an example of an MD document for the `RunbookEntrypoint.ps1` script:
```
Comment (bartduncan) [inline Source/RunbookEntrypoint.md:1]: :D

**Review by bartduncan — APPROVED** [review]

**Pull request merged**